packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js

```diff
@@ -111,7 +125,7 @@ const cleanData = ({ browserState, serverState }, currentSchema, componentsSchem
 
           cleanedData = {
             disconnect: disconnectedRelations,
-            connect: connectedRelations,
+            connect: connectedRelations.reverse(),
```

Comment:
I think this bit deserves a code comment. What do you think?

Reply:
Yeah you're probably right. I almost wonder if it's worth changing back until BE decide their final API implementation? 🤔

Reply:
I'd leave it for now.